packages/command-palette/src/commands/use-single-site-commands.tsx

```diff
@@ -528,15 +530,25 @@ const useSingleSiteCommands = ( { navigate, currentRoute }: useCommandsParams ):
 				),
 				_x( 'audit log', 'Keyword for the Open activity log command', __i18n_text_domain__ ),
 			].join( ' ' ),
-			callback: commandNavigation( '/activity-log/:site' ),
+			callback: commandNavigation(
+				`${
+					shouldUseWpAdmin
+						? 'https://jetpack.com/redirect/?source=calypso-activity-log&site='
+						: '/activity-log/'
+				}:site`
+			),
 			filterP2: true,
 			icon: acitvityLogIcon,
 		},
 		{
 			name: 'openJetpackBackup',
 			label: __( 'Open Jetpack Backup', __i18n_text_domain__ ),
 			callback: commandNavigation(
-				`${ siteType === SiteType.ATOMIC ? 'https://cloud.jetpack.com' : '' }/backup/:site`
+				`${
+					shouldUseWpAdmin
+						? 'https://jetpack.com/redirect/?source=calypso-backups&site='
```

Comment:
This redirects me to Calypso (`https://wordpress.com/backup/:site`). I was expecting a redirection tp WP Admin.

I tested with an Atomic site and the "classic interface" setting enabled.

Reply:
I cannot reproduce this. 🤔 

Are you sure you have run `install-plugin.sh command-palette untangle/cmd-palette` and you're sandboxing `widgets.wp.com`?

Reply:
I tested the direct URL in the browser. If I go to `https://jetpack.com/redirect/?source=calypso-backups&site=testmmtr2024atomic.wpcomstaging.com`, I'm redirected to `https://wordpress.com/backup/testmmtr2024atomic.wpcomstaging.com`, despite having the classic interface selected in that site:

https://github.com/Automattic/wp-calypso/assets/1233880/1321b93b-f5eb-4712-bcf4-e16841e7901c

Reply:
Ahh, sorry! I forgot that Jetpack Cloud is currently only enabled to Atomic classic sites with the early release option set. You can set it in your site's _cli with this command: `wp option update wpcom_classic_early_release 1` and try again 😄

Reply:
Do we need then to add a cloud.jetpack.com fallback redirect for sites that don't have the early release option set?

Reply:
The fallback is `wp.com/backup/:site`, which is already working in this case.

Reply:
Ah, I see the confusion.

> which works for my Atomic site

We are now restricting the new untangled experience to sites with the `wpcom_classic_early_release` blog option set. When this is live (soon-ish), Jetpack Cloud will not work for classic Atomic sites without the above option. Such sites will still need to go to `wp.com/backup`.

Jetpack redirection will magically fall back to `wp.com/<mapped path>` if an ineligible sites try to access Cloud.

Summary:

- For "still-entangled" sites, we send them to `wp.com/backup`.
- For "untangled" ones, we send them to `cloud.jetpack.com/backup/:site`.

Reply:
I updated the table in the PR description to better reflect the scenarios. Sorry for the confusion!